docs/source/conf.py

```diff
@@ -1,26 +1,258 @@
+#!/usr/bin/env python
+#
+# pyesmda documentation build configuration file, created by
```

Comment:
Remove reference to `pyesmda` please :) Perhaps we should run this setup again, it if it was run in 2017 (many years ago)?

Reply:
Fixed. I cleaned up the conf.py